android/src/main/cpp/AndroidLogger.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <string>
```

Comment:
Let's add a newline here to separate different types of headers (we do it often in our codebase).
```suggestion
#include <string>

```